functions/helpFunctions.tsx

```diff
@@ -1,19 +1,42 @@
+import type { TextboxDataType } from '../types/Types';
+
 export default function cleanHtmlText(content:string) {
```

Comment:
does this function need to be exported now that it is only used in this file?

Reply:
Dont think so, no!